script/update_jito_validators.rb

```diff
@@ -18,9 +18,17 @@
                                    AND vote_accounts.account IN (?)
                                    AND vote_accounts.is_active = TRUE",
                                   "mainnet",
-                                  jito_vote_accounts
+                                  jito_vote_accounts.map { |jito_vote_account| jito_vote_account[0] }
                                 )
 
-  jito_db_validators.update_all(jito: true)
+  jito_db_validators.each do |validator|
+    validator.update(
+      jito_commission: jito_vote_accounts.find { |jito_vote_account| 
+        jito_vote_account[0] == validator.vote_account_active.account 
+      }[1],
```

Comment:
After applying a Hash please do also:
```suggestion
      jito_commission: jito_vote_accounts[validator.vote_account_active.account],
```